### PR TITLE
[Admin] Fix typos.

### PR DESCRIPTION
**Pull request by kbrandl**
Changes **Office Store** -> **AppSource** and **side-load** -> **sideload** throughout the docs, and also fixes code formatting issues in the **On Send** article. Note that I've intentionally *not* updated `ms.date` in any of these articles, since none of the changes are substantive.

### CODE REVIEW

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[1c60fd6](https://github.com/OfficeDev/outlook-dev-docs/pull/646/commits/1c60fd6689bdeff3841a715232bcd75508666aa8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/actionable-messages/invoke-add-in.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/actionable-messages/invoke-add-in.md) | :white_check_mark:Succeeded | |
[docs/add-ins/add-in-requirements.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/add-in-requirements.md) | :white_check_mark:Succeeded | |
[docs/add-ins/apis.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/apis.md) | :white_check_mark:Succeeded | |
[docs/add-ins/authenticate-a-user-with-an-sso-token.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/authenticate-a-user-with-an-sso-token.md) | :white_check_mark:Succeeded | |
[docs/add-ins/index.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/index.md) | :white_check_mark:Succeeded | |
[docs/add-ins/outlook-mobile-addins.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/outlook-mobile-addins.md) | :white_check_mark:Succeeded | |
[docs/add-ins/outlook-on-send-addins.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/outlook-on-send-addins.md) | :white_check_mark:Succeeded | |
[docs/add-ins/pinnable-taskpane.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/pinnable-taskpane.md) | :white_check_mark:Succeeded | |
[docs/add-ins/privacy-and-security.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/add-ins/testing-and-tips.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/testing-and-tips.md) | :white_check_mark:Succeeded | |
[docs/add-ins/understanding-outlook-add-in-permissions.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/understanding-outlook-add-in-permissions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C17%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201907172247416847-646%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=97t1lUwfealsijVbO7OyhPqZ6bmMgi8dt%2F%2F98TY8OE4%3D&st=2019-07-17T22%3A43%3A40Z&se=2019-08-17T22%3A48%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/actionable-messages/invoke-add-in.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Actionable messages allow the user to take quick actions on an email message or
 `Action.InvokeAddInCommand` actions can work with add-ins that are already installed by the user, or they can work with add-ins that are not installed. If the required add-in is not installed, the user is prompted to install the add-in with a single click.
 
 > [!NOTE]
-> Single-click installation of the required add-in is only supported if the add-in is published in the [Office Store](https://docs.microsoft.com/office/dev/store/submit-to-the-office-store).
+> Single-click installation of the required add-in is only supported if the add-in is published in the [AppSource](https://docs.microsoft.com/office/dev/store/submit-to-the-office-store).
```
Comment (ElizabethSamuel-MSFT) [inline docs/actionable-messages/invoke-add-in.md]:
```suggestion
> Single-click installation of the required add-in is only supported if the add-in is published in [AppSource](/office/dev/store/submit-to-the-office-store).
```

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[ecc5244](https://github.com/OfficeDev/outlook-dev-docs/pull/646/commits/ecc5244bf5fa89b1857f75fcc6e88e02a0e784d9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/actionable-messages/invoke-add-in.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/actionable-messages/invoke-add-in.md) | :white_check_mark:Succeeded | |
[docs/add-ins/add-in-requirements.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/add-in-requirements.md) | :white_check_mark:Succeeded | |
[docs/add-ins/apis.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/apis.md) | :white_check_mark:Succeeded | |
[docs/add-ins/authenticate-a-user-with-an-sso-token.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/authenticate-a-user-with-an-sso-token.md) | :white_check_mark:Succeeded | |
[docs/add-ins/index.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/index.md) | :white_check_mark:Succeeded | |
[docs/add-ins/outlook-mobile-addins.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/outlook-mobile-addins.md) | :white_check_mark:Succeeded | |
[docs/add-ins/outlook-on-send-addins.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/outlook-on-send-addins.md) | :white_check_mark:Succeeded | |
[docs/add-ins/pinnable-taskpane.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/pinnable-taskpane.md) | :white_check_mark:Succeeded | |
[docs/add-ins/privacy-and-security.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/add-ins/testing-and-tips.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/testing-and-tips.md) | :white_check_mark:Succeeded | |
[docs/add-ins/understanding-outlook-add-in-permissions.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-update-on-send-article/docs/add-ins/understanding-outlook-add-in-permissions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C17%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201907172255210477-646%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=whnZ6yu5AS%2BEFvdp3Zo2%2Bhe6omL0g5k%2BUWFvPwj2h%2Fo%3D&st=2019-07-17T22%3A51%3A12Z&se=2019-08-17T22%3A56%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kbrandl** [comment]
@ElizabethSamuel-MSFT thanks for the quick review and feedback!

**Pull request merged**